polkadot/node/network/approval-distribution/src/lib.rs

```diff
@@ -926,9 +943,14 @@ impl State {
 		.await;
 	}
 
-	async fn process_incoming_assignments<Context, R>(
+	async fn process_incoming_assignments<
+		N: overseer::SubsystemSender<NetworkBridgeTxMessage>,
+		A: overseer::SubsystemSender<ApprovalVotingMessage>,
+		R,
```

Comment:
nit: We already have a `where` clause here. We should move all bounds there

Reply:
Done.